### PR TITLE
reinstate space just for premium label

### PR DESCRIPTION
**Pull request by Thurst-ft**
Reverts the change made in https://github.com/Financial-Times/x-dash/pull/282 with as slight amend so space is only applied when a premium content label is required.

Fixes part of the bug raised here: https://trello.com/c/CkoIA7pZ/889-padding-issue-with-topic-tracker-label

spacing_on_teaser_title

Following a discussion with Lee about amending this in o-tracker it was recommened that the fix was made in x-teaser: https://financialtimes.slack.com/archives/C02FU5ARJ/p1557917172106700

### CODE REVIEW

**Comment by i-like-robots** [comment]
As I understand it (I could be wrong!) I think the solution to this is to ensure the app is using an up-to-date version of o-teaser which implements the correct margins between components.

https://github.com/Financial-Times/o-teaser/pull/119

**Comment by notlee** [comment]
Hey, I helped look into this yesterday. Looks like those changes apply to tag elements `o-teaser__tag`, `o-teaser__tag-prefix`, and `o-teaser__tag-suffix` (previously o-teaser__duration) but labels are applied independent of tags.

'o-teaser' demos output [promoted content labels](https://www.ft.com/__origami/service/build/v2/demos/o-teaser@3.5.3/promoted-content?brand=master) in a span and rely on the space. We should probably output premium labels as part of `o-teaser` like we do with promoted content labels `o-teaser__promoted-prefix`, but that's a separate issue.

File: components/x-teaser/src/Title.jsx [hunk 1]
```diff
@@ -16,8 +16,11 @@ export default ({ title, altTitle, headlineTesting, relativeUrl, url, indicators
 				{displayTitle}
 			</Link>
 			{indicators && indicators.accessLevel === 'premium' ? (
-				<span className={premiumClass} aria-label="Premium content">
-					Premium
+				<span>
```
Comment (i-like-robots) [inline components/x-teaser/src/Title.jsx]: Is the outer span required?

**Comment by i-like-robots** [comment]
Ah, I'm with you, thanks for clarifying @notlee.

File: components/x-teaser/src/Title.jsx (hunk 1 above)
Comment (Thurst-ft) [reply components/x-teaser/src/Title.jsx]: It seems so, throws an error without, which I think is to do with jsx not allowing adjsent elements.

Comment (i-like-robots) [reply components/x-teaser/src/Title.jsx]: Ah, boo.

Comment (quarterto) [reply components/x-teaser/src/Title.jsx]: you can in React with a `<Fragment>`, but that's not supported in stable Preact, so we can't use it in x-dash yet

Comment (Thurst-ft) [reply components/x-teaser/src/Title.jsx]: @i-like-robots happy to take a recommendation on a better way.

Comment (i-like-robots) [reply components/x-teaser/src/Title.jsx]: It's fine the way it is, no need to delay!

**Pull request merged**